eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -75,22 +75,6 @@ steps:
     displayName: Reset Docs Repo, Checkout Onboarding Branch
     ignoreLASTEXITCODE: false
 
-  - task: PowerShell@2
-    displayName: 'Update Docs.MS CI Targeted Packages'
-    inputs:
-      targetType: filePath
-      filePath: ${{ parameters.ScriptDirectory }}/update-docs-ci.ps1
-      arguments: >
-        -ArtifactLocation ${{ parameters.ArtifactLocation }}
-        -WorkDirectory "${{ parameters.WorkingDirectory }}"
-        -RepoId ${{ parameters.RepoId }}
-        -Repository ${{ parameters.PackageRepository }}
-        -ReleaseSHA ${{ parameters.ReleaseSha }}
-        -DocRepoLocation "${{ parameters.WorkingDirectory }}/repo"
-        -Configs "${{ parameters.CIConfigs }}"
-      pwsh: true
-    env:
-      GH_TOKEN: $(azuresdk-github-pat)
   - template: /eng/common/pipelines/templates/steps/git-push-changes.yml
```

Comment:
Are the other steps needed after removing this? I suspect the others will always be a noop now.

Reply:
@danieljurek Is the onboarding branch still needed once we want to update the package list from release csv?

Reply:
Testing in .NET template

Reply:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=838947&view=logs&j=5ccebfdc-da53-51d4-fecb-9480b00e8628&t=5ccebfdc-da53-51d4-fecb-9480b00e8628